journalbeat/plan.sh

```diff
@@ -0,0 +1,35 @@
+pkg_name=journalbeat
+pkg_origin=core
+pkg_version="6.6.0"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_deps=(core/glibc core/systemd)
+pkg_build_deps=(core/go core/git core/make core/gcc core/systemd core/patchelf)
+pkg_bin_dirs=(bin)
+pkg_svc_user=root
+pkg_svc_group=root
+pkg_description="Journalbeat is a lightweight journald log shipper for Elasticsearch."
+pkg_upstream_url="https://github.com/elastic/beats/tree/master/journalbeat"
+
+do_download() {
+  return 0
+}
+
+do_unpack() {
+  return 0
+}
+
+do_build() {
+  SYSTEMD_LIB_PATH=$(pkg_path_for core/systemd)
+  GOPATH="$(dirname "${HAB_CACHE_SRC_PATH}")"
+  export GOPATH
+  CGO_CFLAGS="-I${SYSTEMD_LIB_PATH}/include" go get github.com/elastic/beats/journalbeat
```

Comment:
Nothing to change here,   but I tried `CGO_FLAGS="${CFLAGS}"` (and `${CFLAGS[@]}`) since CFLAGS includes  `-I /path/to/systemd/include`  but it failed to build. My guess is something to do with multiple -I in CFLAGS or the expected separator is different, but I don't have the golang expertise to make a recommendation.